### ISSUE TITLE
Conversion from .pb file to .dlc format

### ISSUE THREAD

**Issue by himsikha123** [tensorflow/haskell#170]
I am trying to convert MNIST.pb file to .dlc format. But i am getting the following error :
1.
(tensorflow)root@pcz-ee210189:/u/snpe-1.2.2_android/models/mnist# snpe-tensorflow-to-dlc --graph /u/snpe-1.2.2_android/models/mnist/MNIST.pb  --input_dim Mul 28,28,3 --dlc mnist_dlc.dlc --out_node softmax

error:
2017-12-08 12:51:58,788 - 122 - ERROR - Conversion failed: ERROR_TF_NODE_NOT_FOUND_IN_GRAPH: Node not found in graph. Node name: Mul

Please guide me regarding the input _dimension.

Tensorflow version:1.4.0
python:2.7

**Comment by fkm3** [comment]
I think you've created this issue in the wrong repo, try https://github.com/tensorflow/tensorflow/issues instead.
